clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -395,21 +394,6 @@ static CIRGenCallee buildDirectCallee(CIRGenModule &CGM, GlobalDecl GD) {
   return CIRGenCallee::forDirect(CalleePtr, GD);
 }
 
-// TODO: this can also be abstrated into common AST helpers
-bool CIRGenFunction::hasBooleanRepresentation(QualType Ty) {
```

Comment:
Instead of removing this function, which will still be valuable to us (e.g. we need to properly handle atomics), we should just change the faulty caller, see my other comment.